### ISSUE TITLE
Crash when placing image attachments

### ISSUE THREAD

**Issue by Pajn** [Foundry376/Mailspring#1446]
##### Are there any related issues?

I could not find any

##### What operating system are you using?

Ubuntu 19.04

##### What version of Mailspring are you using?

1.6.0

--

**Bug?**

I drag and dropped three images into the mail conversation and began creating newlines around them for spacing when the editor crashed with:

```
Error: Failed to execute 'setStart' on 'Range': The offset 4294967294 is larger than the node's length (1).
    at findDOMRange (/usr/share/mailspring/resources/app.asar/node_modules/slate-react/lib/utils/find-dom-range.js:37:5)
    at Content._this.updateSelection (/usr/share/mailspring/resources/app.asar/node_modules/slate-react/lib/components/content.js:154:46)
    at Content._this.componentDidUpdate (/usr/share/mailspring/resources/app.asar/node_modules/slate-react/lib/components/content.js:121:13)
    at Hi (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:227:383)
    at Fi (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:218:212)
    at Gi (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:217:174)
```

##### Is this an email sync or connection issue?

No

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?

No

##### Is the issue reproducible with a particular attachment, message, signature, etc?

I can not share the specific images unfortunately.

--

**Feature Request?**

##### Does this feature exist in another mail client or tool you use?

...

**Comment by nickallevato** [comment]
This also happens to me.

**Comment by bengotow** [comment]
ooof. man, this is a bad one. We're upgrading the SlateJS text editor in the next release (I just have to fix a few more things and we can start testing it.) Will make sure I get that done ASAP, I think it'll resolve this once and for all.

**Comment by bengotow** [comment]
Hey folks! This should be addressed in the latest release (1.6.3) that shipped earlier this week. The app should auto-update on it's own, or you can grab the new version from the releases page: https://github.com/Foundry376/Mailspring/releases/. Thanks for filing this - enjoy the update!
